### DISCUSSION TITLE
希望能增加在线播放时候缓冲到底或者缓冲时长设置的功能

### DISCUSSION THREAD

**Discussion by TonyShitDark** [the1812/Bilibili-Evolved#2300] [功能建议 / Ideas]
如题   希望能增加在线播放时候缓冲到底或者缓冲时长设置的功能
在线看视频时候开了倍速 分片缓冲有时候会卡一下能缓冲到底就好了

**Comment by CKylinMC** [comment]
https://github.com/the1812/Bilibili-Evolved/discussions/2243

好像有人提过这个问题，不过这个功能感觉可能性不大
